cmd/insprd/operators/kafka/channels/client.go

```diff
@@ -48,9 +48,9 @@ func NewOperator(mem memory.Manager) (*ChannelOperator, error) {
 		adminClient = &mockAdminClient{}
 	} else {
 		logger.Info("initializing kafka admin with production configs",
-			zap.String("kafka bootstrap servers", getEnv().kafkaBootstrapServers))
+			zap.String("kafka bootstrap servers", "kafka.default.svc:9092"))
 		config = &kafka.ConfigMap{
-			"bootstrap.servers": getEnv().kafkaBootstrapServers,
+			"bootstrap.servers": "kafka.default.svc:9092",
```

Comment:
Are these to be hardcoded like this? there is a bootstrapserver variable in the kafka configuration. Not sure tho

Reply:
These configs are been distributed trough env vars, but it shouldn't be hard coded